components/Modal/Modal.spec.tsx

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react'
+import { Modal } from '.'
+
+describe('Modal component', () => {
+    it('renders correctly', () => {
+        render(
+            <Modal isOpen onClose={() => console.log('onClose')}> 
```

Comment:
You can mock the `onClose` function using [jest mock functions](https://jestjs.io/docs/mock-functions)
```
const onClose = jest.fn();
...
            <Modal isOpen onClose={onClose}> 
```

And then test if it was called when user clicks on close (for instance)